src/main/java/com/google/ehub/servlets/LoginServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+  String userEmail;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    if (userService.isUserLoggedIn()) {
+      userEmail = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterUserLogsOut = "/LoginPage.html";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+      response.setContentType("text/html");
+      response.getWriter().println("<p>Hello " + userEmail + "!</p>");
+      response.getWriter().println(logoutUrl);
+    } else { // User is not Loggedin //TODO(oyins): create a "Create Account" button
+<<<<<<< HEAD
+      String urlToRedirectToAfterUserLogsIn = "/CreateProfilePage.html";
+=======
+      String urlToRedirectToAfterUserLogsIn = "/ProfilePage.html";
+>>>>>>> 81a5bc18cba91e7a73a301023a0c5c0d8ebdcca9
+      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+
```

Comment:
Looks like you need to fix the merge conflicts for each file.

Reply:
fixed